assets/components/directDebit/directDebitForm/directDebitForm.jsx

```diff
@@ -312,6 +312,8 @@ function PaymentButton(props: {
       </span>
     );
   }
+
+  return null;
```

Comment:
Can we add `let resp = null;` variable  at the top and only returns once at the bottom in this function?

Reply:
Yeah, I think both are correct, since there are only two possible option I personally feel more suitable:
```
let resp = null;
if(props.phase === 'case a') {
 resp = (<...>)
} else if(props.phase === 'case b')  {
 resp = (<...>) 
}
return resp
```
but the switch statement is more extensible..

Reply:
I think either an `if` of the form:

```jsx
if (thing === One) {
  return x;
} else if (thing === Two) {
  return y;
}

return z;
```

or a `switch` of the form:

```jsx
switch (thing) {
  case One:
    return x;
  case Two:
    return y;
  default:
    return z;
}
```

works here. @akash1810 I agree that the switch is a bit easier to read.